m2cgen/interpreters/elixir/code_generator.py

```diff
@@ -0,0 +1,62 @@
+from contextlib import contextmanager
+
+from m2cgen.interpreters.code_generator import CodeTemplate, FunctionalCodeGenerator
+
+
+class ElixirCodeGenerator(FunctionalCodeGenerator):
+    tpl_function_signature = CodeTemplate("{function_name} = fn ->")
+    tpl_if_statement = CodeTemplate("cond do {if_def} -> ")
+    tpl_else_statement = CodeTemplate("true ->")
```

Comment:
Why did you choose `cond` and not `if/else`?

Reply:
It was a suggestion from José Valim here: https://github.com/elixir-nx/nx/issues/435#issuecomment-894673035

"Also, consider using cond to avoid some of the nesting, which should also make the compiler happier."



I'm pretty sure `if/else` is a macro, while `cond` is a language feature.

Reply:
Ah, I see! Sorry for missing that response.